src/sentry/static/sentry/less/shared-components.less

```diff
@@ -2724,6 +2724,9 @@ ul.radio-inputs {
     padding: 20px 30px;
     border-top: 1px solid @trim;
     margin: 20px -30px -30px;
+    display: flex;
+    justify-content: flex-end;
+    text-align: initial;
```

Comment:
`text-align` is an inherited property and therefore likely to cause bugs. What exactly is this overriding?

Reply:
Because `text-align` gets set to `right` by the bootstrap class.

Reply:
👍